extensions/src/tables/index.ts

```diff
@@ -166,6 +198,51 @@ export default class Tables extends Extension<Details, Blocks> {
             newRow.push(0);
           }
           this.tables[table].push(newRow);
+          this.rowNames[table].push('row');
+        }
+      }),
+      // removes the specified column from the table
+      deleteColumn: (self: Tables) => ({
+        type: BlockType.Command,
+        args: [self.tableNamesArg, ArgumentType.Number],
+        text: (table, column) => `delete column ${column} from ${table}`,
+        operation: (table, column) => {
+          if (!(table in self.tables)) {
+            alert(`that table does not exist.`);
+            return;
+          }
+          if (
+            column > this.tables[table][0].length || 
+            column < 0
+          ) {
+            alert(`that column number doesn't exist.`);
+            return;
+          }
+          for (let i = 0; i < this.tables[table].length; i++) {
+            this.tables[table][i].splice((column - 1), 1);
+          }
+          this.columnNames[table].splice((column - 1), 1);
+        }
+      }),
+      // removes the specified row from th table
+      deleteRow: (self: Tables) => ({
+        type: BlockType.Command,
+        args: [self.tableNamesArg, ArgumentType.Number],
```

Comment:
Since rows index from 1 (correct?) I think it be good to give these arguments a default value of `1` (instead of `0`)